Foundation/src/Thread_POSIX.cpp

```diff
@@ -122,7 +123,8 @@ void ThreadImpl::setNameImpl(const std::string& threadName)
 	}
 
 	ScopedLock<FastMutex> lock(_pData->mutex);
-	if (realName != _pData->name) {
+	if (realName != _pData->name)
+    {
```

Comment:
well, now we have the braces at the right place, but spaces indentation instead of tab. Please, do read the [Coding Style Guide](https://www.appinf.com/download/CppCodingStyleGuide.pdf)

Reply:
I forgot set tab as indent in my new project settings.